### PR TITLE
Minor text edits

### PR DESCRIPTION
**Pull request by iadawn**
Not sure about the 'and style sheets'. I see style sheets as the way to effect alternative presentations. I was also concerned that it might suggest teaching designers CSS which may not be the intention

### CODE REVIEW

**Comment by carlosapaduarte** [comment]
My 2 cents on the style sheets: it's important that designers are aware of user style sheets. I suppose that was Daniel's intention.

**Comment by iadawn** [comment]
Agree that they should be aware, I wonder if this is a bit creeping up on them. Will see if I can re-read with that thought in mind.

File: content/designer/visual-design.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Students should be able to:
 * determine sufficient color contrast for text, images of text, and controls
 * design text and images of text that have a color contrast of at least 4.5:1 with respect to their background
 * design large-scale text and images of text that have a color contrast of at least 3:1 with respect to their background
-* design components and graphics that have a color contrast of at least 3:1 with respect to their background
+* design user interface components and graphics that have a color contrast of at least 3:1 with respect to their background
```
Comment (daniel-montalvo) [inline content/designer/visual-design.md]: Thanks @iadawn 

We had "user interface components" before. Some of the learning outcomes started with "design user interfaces that ..." My understanding is that phrase made most of the learning outcomes very long and difficult to process. Why do you think "user interface components" is needed here?

Comment (iadawn) [reply content/designer/visual-design.md]: I think it was the 'components and graphics' got me thinking of 'graphics' as 'images' and that isn't something that would be covered by this. Trying to be a bit more specific in this instance helped it in my head.

**Pull request merged**